charts/argo-cd/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: v2.7.0-cap-CR-18361-custom-instance-label
+appVersion: v2.7.0-cap-CR-19754-deletion-fix
 kubeVersion: ">=1.23.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 5.38.1-1-cap-CR-18361
+version: 5.38.1-2-cap-CR-19754
```

Comment:
I'm not sure I understand what changed here?

Reply:
You need to change appVersion @yaroslav-codefresh